src/content/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to browser monitoring
+title: Using New Relic for browser monitoring
```

Comment:
Would you also change the browser.yml title to "Using new relic for browser monitoring"?

Reply:
Good call! I'll get it changed